source/transactions/transactions.rst

```diff
@@ -1283,8 +1283,8 @@ The following commands are allowed inside transactions:
 
 8.  aggregate (including $lookup)
 
-    -  The $out stage is prohibited because it uses collection create
-       and rename operations.
+    -  The ``$out`` and ``$merge`` stages are prohibited because they use
+       collection create and rename operations.
```

Comment:
Based on a local test with MongoDB 4.1.13, a pipeline with `$merge` is causes `aggregate` to fail with code 263 ("OperationNotSupportedInTransaction") and message "$merge cannot be used in a transaction". This error occurs regardless of whether the collection exists prior to running the aggregation, so I assume the server is just preemptively rejecting the pipeline rather than waiting to see if it would attempt to create a new collection.

Reply:
I'm not sure if the $merge stage runs create/rename ops. Maybe we should just say they are prohibited and that's it:
```
-  The ``$out`` and ``$merge`` stages are prohibited.
```